src/assets/stylesheets/index.scss

```diff
@@ -1756,3 +1756,40 @@ header.portal-only section:after {
   font-family: bootstrap-icons !important;
   content: '\f285';
 }
+
+/* contrast mode - CSS for more contrast
+ =================================================================== */
+@media (prefers-contrast: more) {
+  :root {
+    --light-grey: #eee;
+    --middle-grey: #e3e3e3;
+  }
+  main {
+    background: var(--light-grey);
+  }
+  .Toastify__toast,
+  .navbar-toggler.btn,
+  .form-control,
+  .form-select,
+  .card,
+  .editor-img,
+  .box-comment .avatar img,
+  .box-attachment {
+    border: 1px solid #777;
```

Comment:
I found one place, where the border is not (yet) working nicely: In the `DataMessageWidget` - the arrow to the avatar should also have a border:

<img width="1103" alt="Screenshot 2024-03-01 at 12 12 08" src="https://github.com/Scrivito/scrivito-portal-app/assets/86275/883d2122-a3a9-4d63-8d41-cefa726a35eb">